main.go

```diff
@@ -40,6 +40,7 @@ func main() {
 		collapseRoot    bool
 		onlyFolders     bool
 		colorBlind      bool
+		percentCoverage bool
```

Comment:
symbol names match command line option names for consistency
```suggestion
		percent bool
```